demo-shell/src/app/components/cloud/service-task-list-cloud-demo.component.html

```diff
@@ -0,0 +1,29 @@
+<div fxLayout="column" fxFill fxLayoutGap="2px">
+    <adf-cloud-edit-service-task-filter
+        [id]="'myFilter'"
+        [filterProperties]="taskFilterProperties.filterProperties"
+        [sortProperties]="taskFilterProperties.sortProperties"
+        [actions]="taskFilterProperties.actions"
+        (filterChange)="onFilterChange($event)">
+    </adf-cloud-edit-service-task-filter>
+    <div fxLayout="column" fxFlex fxLayoutAlign="space-between" *ngIf="editedFilter">
```

Comment:
same as above, you don't really need flex-layout directives here, can be done via pure css